src/components/layout/GenericLayout/index.tsx

```diff
@@ -1,19 +1,41 @@
 import React from 'react'
-import { Header } from './Header'
+
+import ThemeProvider, { StaticGlobalStyle, ThemedGlobalStyle } from 'theme'
+
 import { Footer } from './Footer'
-import { StaticGlobalStyle } from 'theme'
-export interface Props {
-  menu?: React.ReactNode
-  navTools?: React.ReactNode
+import { Header } from './Header'
+
+export type Props = {
+  header?: JSX.Element | null
+  footer?: JSX.Element | null
+  children: React.ReactNode
```

Comment:
don't need this, FC passes children implicitly. could be a linter issue if you're seeing errors

Reply:
Yeah, the storybook gets angry when I don't tell it has a `children` prop
I can use the `PropsWithChildren` though (I found out about this later)

Reply:
hmmm okidoki